documentation/book/con-scram-sha-authentication.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+
+[id='con-scram-sha-authentication-{context}']
+= SCRAM-SHA authentication
+
+Kafka can be configured to use SASL SCRAM-SHA to provide authentication on both unencrypted and TLS connections. 
+When used with a TLS connection, the TLS protocol is providing encryption, but not authentication.
+
+SCRAM (Salted Challenge Response Authentication Mechanism) is an authentication protocol that can establish mutual authentication using passwords. SCRAM has some useful properties which mean it is considered safe to use SCRAM-SHA even on unencrypted connections:
+
+* The passwords are not sent in the clear over the communication channel.
+Instead the client and the server are each challenged by the other to offer proof that they know the password of the authenticating user.
+
+* The server and client each generate a new challenge one each authentication exchange. 
+This means that the exchange is resilient against replay attacks.
+
+== When to use SCRAM-SHA authentication for clients
+
+SCRAM-SHA is a suitable choice for authenticating Kafka clients when:
+
+* The client supports authentication using SCRAM-SHA-512
```

Comment:
This is the first point where I read 512. Before that we are talking about scram-sha in general without mentioning that we support only the 512 version; now we are introducing it as a client preference. Should we mentioned that before?

Reply:
It's not a client preference. We only support 512. Obviously the client needs to support what we support, so it's worth saying what that is.

Reply:
What I meant is that we mention about 512 only now talking about clients. I think that we should mention that we support only the 512 version even before.